framework/src/Volo.Abp.MongoDB/Microsoft/Extensions/DependencyInjection/AbpMongoDbServiceCollectionExtensions.cs

```diff
@@ -11,13 +13,20 @@ public static class AbpMongoDbServiceCollectionExtensions
             where TMongoDbContext : AbpMongoDbContext
         {
             var options = new AbpMongoDbContextRegistrationOptions(typeof(TMongoDbContext), services);
+
+            var replaceDbContextAttribute = typeof(TMongoDbContext).GetCustomAttribute<ReplaceDbContextAttribute>(true);
```

Comment:
Get all ReplaceDbContextAttribute attributes, not single one. We may use like that:

````csharp
[ReplaceDbContextAttribute(typeof(ISomeDbContext))]
[ReplaceDbContextAttribute(typeof(IOtherDbContext))]
public class MyDbContext...
{
}
````

It should replace all.